latexmkrc

```diff
@@ -1,6 +1,10 @@
 if ($ENV{HOMM3_PRINTABLE}) {
   &alt_tex_cmds;
   $pre_tex_code = '\AtBeginDocument{\toggletrue{printable}}';
+  
+  if ($ENV{HOMM3_NO_ART_BACKGROUND}) {
```

Comment:
As Alexey suggested, it can be an independent switch.

Reply:
See https://github.com/Heegu-sama/Homm3BG/pull/553#issuecomment-2145471886